tasks/docker.yml

```diff
@@ -1,9 +1,73 @@
 ---
-- name: Installing docker
+- name: Installing Python docker module requirements
```

Comment:
Yes Installing docker is confusing. Well catch.
Installing docker SDK? or docker SDK for Python

Reply:
This is the Python library for the Docker Engine API (installing via pip). This pip module is needed for Ansible to use the docker.container module, so I think it makes sense to install it before using docker with Ansible.

Reply:
My comment was only about the task name